deploy-service/common/src/main/java/com/pinterest/deployservice/handler/CommonHandler.java

```diff
@@ -398,10 +398,12 @@ public void transitionDeployState(String deployId, EnvironBean envBean) throws E
         String lockName = String.format("STATE_TRANSITION-%s", deployId);
         Connection connection = utilDAO.getLock(lockName);
         if (connection != null) {
+            LOG.debug(String.format("Successfully get lock: %s", lockName));
```

Comment:
the prod env set the log level as INFO.  i think for lock msg, it might be ok to use LOG.info for the current critical debugging.

Reply:
we can change the log level to be DEBUG when we need debugging lock related issues. get/release locks happen frequently. currently teletraan already generates a large amount of INFO logs, that's why i use DEBUG level for these logs.

Reply:
my view is we are in active debugging of the lock leak issue so we can make it info for now and once stable then turn it to debug. and across the system there are some other place (see the other file in the PR) has the lock msg DEBUG.  and it will generally take some time to identify the leak so turning on DEBUG for prod temporarily might not be sufficient and need to restart for log level change.